src/com/facebook/buck/jvm/kotlin/KotlinBuckConfig.java

```diff
@@ -20,26 +20,102 @@
 import com.facebook.buck.io.ExecutableFinder;
 import com.facebook.buck.rules.HashedFileTool;
 import com.facebook.buck.rules.Tool;
+import com.facebook.buck.util.HumanReadableException;
+
```

Comment:
nit: no space. doesn't look like other files have a newline between com.facebook and com.google